contracts/KetlAttestation.sol

```diff
@@ -81,7 +81,10 @@ contract KetlAttestation is ERC1155, Ownable, Versioned, ERC2771Recipient {
   mapping(uint => IncrementalTreeData) public entanglementsTrees;
   mapping(uint => uint[]) public entanglements;
   mapping(uint => mapping(uint => bool)) public entanglementsRoots;
-  mapping(uint => bool) public attestationHashesEntangled;
+  // Attestations to total number of entanglements
+  mapping(uint => Counters.Counter) public attestationHashesEntangled;
+  mapping(uint => uint16) public maxEntanglementsPerAttestation;
+  uint16 public globalMaxEntanglementsPerAttestation = 1;
```

Comment:
I think this is unnecessary, if it's 0 let's leave it as 0 — and write less things in the smart contract

Reply:
We can change the logic to allow entanglements if `attestationHashesEntangled` == 0, so all attestationHashes can by default be used once (without this global variable). 

The current logic would not allow users to `registerEntanglement` unless we set `maxEntanglementsPerAttestation` to 1 or more first. I assume we do not want to set this value for every single attestationHash.